ProjectLighthouse.Servers.API/Controllers/RpcController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using LBPUnion.ProjectLighthouse.Configuration;
+using LBPUnion.ProjectLighthouse.Servers.API.Responses;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LBPUnion.ProjectLighthouse.Servers.API.Controllers;
+
+public class RpcController : ApiEndpointController
+{
+    /// <summary>
+    /// Returns basic information that Discord RPC clients can use for self-configuration.
+    /// </summary>
+    /// <returns>RpcResponse</returns>
+    /// <response code="200">The RPC configuration.</response>
+    [HttpGet("rpc")]
+    [ProducesResponseType(typeof(RpcResponse), StatusCodes.Status200OK)]
+    public IActionResult GetRpcConfiguration()
```

Comment:
should this explicitly specify that it outputs application/json or does MVC do that automatically?

Reply:
MVC will do this automatically